lnst/Tests/Iperf.py

```diff
@@ -34,57 +35,52 @@ def run(self):
 
         try:
             stdout, stderr = server.communicate()
-            stdout = stdout.decode()
-            stderr = stderr.decode()
         except KeyboardInterrupt:
-            pass
+            server.send_signal(signal.SIGINT)
+            stdout, stderr = server.communicate()
```

Comment:
should this be called for other exceptions/signals as well? in case some other issue happens it's probably still useful to try to retrieve the stdout/stderr values.

Reply:
at that point i guess it should just be a non-blocking attempt at retrieving the data...